lib/opentelemetry/trace/exporter/otlp.lua

```diff
@@ -96,12 +96,59 @@ function _M.export_spans(self, spans)
             }
         }
     }
-    for _, span in ipairs(spans) do
+    local tracers = {}
+    local providers = {}
+    tracers[spans[1].tracer] = 1
+    providers[spans[1].tracer.provider] = 1
+    for i, span in ipairs(spans) do
+        local rs_idx = providers[span.tracer.provider]
+        local ils_idx = tracers[span.tracer]
```

Comment:
what is `ils` short for here?

Reply:
instrumentation library, maybe this should become `il_idx` and `rs_idx` should become `r_idx`

Reply:
oh because rs is for resource spans and ils is for instrumentation library spans so this makes sense